src/core/Memory.test.ts

```diff
@@ -1,12 +1,23 @@
-import {getValues, saveValues} from "./Memory";
+import {getValues, saveValues, storageKey} from "./Memory";
+
+const valuesArrMock = [{date: new Date(), value: 21}, {date: new Date(), value: 48}];
+
+beforeEach(() => {
+    localStorage.clear();
+})
 
 test("Test That Value Retrieved will Be Empty If not Set", () => {
     expect(getValues()).toEqual([]);
 })
 
-test("Test that Value Can Be Saved And Retrieved", () => {
-    const value = [{date: new Date(), value: 21}, {date: new Date(), value: 48}];
-    saveValues(value);
-    expect(getValues()).toEqual(value);
+test('When saveValues is executed Then data should be saved in localstorage', () => {
+    saveValues(valuesArrMock);
```

Comment:
short names XD

btw. I think that a better name is CORRECT_VALUE_TO_SAVE.
SNAKE_CASE is for constants.